src/runner/provider/qti.js

```diff
@@ -48,10 +49,15 @@ var qtiItemRuntimeProvider = {
             {
                 assetManager: this.assetManager
             },
-            _.pick(this.options, ['themes', 'preload'])
+            _.pick(this.options, ['themes', 'preload', 'view'])
         );
 
-        this._renderer = new QtiRenderer(rendererOptions);
+        if(rendererOptions.view === 'scorer') {
```

Comment:
is this type accurate? should not it be something like `review` or `readonly` instead?

Reply:
According to IMS standards it is supposed to be called view with the following values:

- author
- candidate
- proctor
- scorer
- tutor 
- testConstructor

https://www.imsglobal.org/question/qtiv2p2p2/QTIv2p2p2-ASI-InformationModelv1p0/imsqtiv2p2p2_asi_v1p0_InfoModelv1p0.html#FigEnumeratedListClass_DataModel_View